### ISSUE TITLE
Unable to install–`@anysphere/tiktoken-node-win32-arm64-msvc@0.1.27 failed to resolve`

### ISSUE THREAD

**Issue by Just-Moh-it** [anysphere/priompt#11]
While installing using bun, getting the following error:

```
❯ bun i @anysphere/priompt                                       
[0.04ms] ".env"
bun add v1.0.18 (36c316a2)
error: @anysphere/tiktoken-node-win32-arm64-msvc@0.1.27 failed to resolve
```

Seems like the [package is published](https://www.npmjs.com/package/@anysphere/tiktoken-node-win32-arm64-msvc?activeTab=versions), just not on v0.1.27

**Comment by Just-Moh-it** [comment]
Might be because i'm using bun?

**Comment by sobjornstad** [comment]
I'm getting this using `yarn` as well

**Comment by arvid220u** [comment]
Thanks for reporting! I believe this is fixed in 0.2.0.

**Comment by sreyemnayr** [comment]
Struggling with this as well - took a while to sort out the examples as they're no longer working with the latest. But now I'm stuck with `Error: Cannot find module './node_modules/@anysphere/tiktoken-node-darwin-arm64/tiktoken-node.darwin-arm64.node'` which is definitely there...
